scripts/add_new_provider.py

```diff
@@ -72,6 +81,12 @@ def review_information(provider_info, admin_info, provider_credentials):
             print "3. Provider Credentials deleted"
 
 
+def yes_no_truth(raw_text):
+    if raw_text.lower().strip() == 'yes':
```

Comment:
We could, but we could also change what this context means in the future (I.e. `in ['y','yes',...]`)